code/modules/research/techweb/all_nodes.dm

```diff
@@ -617,8 +619,8 @@
 	display_name = "Cryostasis Technology"
 	description = "Smart freezing of objects to preserve them!"
 	prereq_ids = list("adv_engi", "biotech")
-	design_ids = list("splitbeaker", "noreactsyringe", "cryotube", "cryo_Grenade", "stasis")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000)
+	design_ids = list("splitbeaker", "noreactsyringe", "cryotube", "cryo_Grenade", "stasis", "cryogenicpod", "cryogenicpodcontrol") // SinguloStation13 Edit (Cryogenic freezers change - added cryogenic pods to research)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000) // SinguloStation13 Edit (Cryogenic freezers change - Tech more expensive for new shit)
```

Comment:
Nothing changed?

Reply:
Not sure how this happened; was changed to `5000` in original code, will need to decide if we want to keep the change or remove the comment.

Reply:
No other technology has cryotech as a prerequisite, so I don't think there's a problem with increasing the point cost in response to the added machines

Reply:
```suggestion
	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000) // SinguloStation13 Edit (Cryogenic freezers change - Tech more expensive for new shit)
```